LonaStudio/Preferences/CSColors.swift

```diff
@@ -35,9 +35,9 @@ class CSColors: CSPreferencesFile {
     }
     
     static func parse(css string: String, withDefault defaultColor: NSColor = NSColor.clear) -> CSColor {
-        let match = CSColors.colors.first(where: { $0.id?.uppercased() == string.uppercased() })
+        let match = CSColors.colors.first(where: { $0.id.uppercased() == string.uppercased() })
         
-        return match ?? CSColor(id: nil, name: "Custom color", color: NSColor.parse(css: string) ?? defaultColor, value: string)
+        return match ?? CSColor(id: "custom", name: "Custom color", color: NSColor.parse(css: string) ?? defaultColor, value: string)
```

Comment:
There _might_ be an issue if the `cscolors.json` file has a color with the id `custom`, since then if we save this id somewhere and try to look it up again, we'll get the color from the `cscolors.json` instead. I'm not 100% sure if this is an issue though, so I think this is 👍  for now. Just something to keep in mind.